includes/class-genesis-responsive-slider-admin.php

```diff
@@ -0,0 +1,432 @@
+<?php
+/**
+ * Genesis Responsive Slider Admin.
+ *
+ * @package genesis-responsive-slider
+ */
+
+/**
+ * Creates settings and outputs admin menu and settings page.
+ *
+ * @package genesis-responsive-slider
+ */
+class Genesis_Responsive_Slider_Admin {
+
+	/**
+	 * Constructor.
+	 */
+	public static function init() {
+		add_action( 'admin_init', array( 'Genesis_Responsive_Slider_Admin', 'register_genesis_responsive_slider_maybe_reset_settings' ) );
+		add_action( 'admin_notices', array( 'Genesis_Responsive_Slider_Admin', 'genesis_responsive_slider_notice' ) );
+		add_action( 'admin_menu', array( 'Genesis_Responsive_Slider_Admin', 'genesis_responsive_slider_settings_init' ), 15 );
+		add_filter( 'screen_layout_columns', array( 'Genesis_Responsive_Slider_Admin', 'genesis_responsive_slider_settings_layout_columns' ), 10, 2 );
+	}
+
+	/**
+	 * Reset settings.
+	 */
+	public static function register_genesis_responsive_slider_maybe_reset_settings() {
+
+		register_setting( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD, GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD );
+		add_option( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD, Genesis_Responsive_Slider::genesis_responsive_slider_defaults(), '', 'yes' );
+
+		if ( Genesis_Responsive_Slider::genesis_get_responsive_slider_option( 'reset' ) ) {
+			update_option( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD, Genesis_Responsive_Slider::genesis_responsive_slider_defaults() );
+
+			genesis_admin_redirect( 'genesis_responsive_slider', array( 'reset' => 'true' ) );
+			exit;
+		}
+
+	}
+
+	/**
+	 * This is the notice that displays when you successfully save or reset
+	 * the slider settings.
+	 */
+	public static function genesis_responsive_slider_notice() {
+
+		if ( ( isset( $_GET['reset'] ) && 'true' === $_GET['reset'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.NoNonceVerification
+			echo '<div class="notice notice-success is-dismissible">' . esc_html( __( 'Settings reset.', 'genesis-responsive-slider' ) ) . '</div>';
```

Comment:
```suggestion
			echo '<div class="notice notice-success is-dismissible"><p>' . esc_html( __( 'Settings reset.', 'genesis-responsive-slider' ) ) . '</p></div>';
```

Notices need to be wrapped in `<p>` tags here, otherwise the styling looks wrong:

<img width="1225" alt="Screenshot 2019-03-12 at 14 09 53" src="https://user-images.githubusercontent.com/647669/54202826-27513d80-44d1-11e9-9d24-2741deb45089.png">